retorchfiles/scripts/coilifecycles/coi-setup.sh

```diff
@@ -1,35 +1,47 @@
 #!/bin/bash
+
```

Comment:
The common steps for COI set-up (further discussion):
SETUP
- Common:
  - Change permissions for the scripts folder.
  - Clean the workspace.
  - Remove older video logs from Selenoid (can be improved).
  - Create the necessary output folders for all TJobs.
  - Pull Selenoid images (to avoid starting testing without VNC or the video recorder).
  - Measure time.
- Specific:
  - Pull necessary images and create images of eShopContainers.
TEAR-DOWN
- Common:
  - Remove all containers and volumes associated with the application.
  - Prune all volumes.
  - Measure time.

Reply:
let talk tomorrow